test/gst-msdk/vpp/csc.py

```diff
@@ -0,0 +1,39 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+
+spec = load_test_spec("vpp", "csc")
+
+@slash.requires(have_gst)
+@slash.requires(*have_gst_element("msdk"))
+@slash.requires(*have_gst_element("msdkvpp"))
+@slash.requires(*have_gst_element("checksumsink2"))
+@slash.requires(using_compatible_driver)
+@slash.parametrize(*gen_vpp_csc_parameters(spec))
+@platform_tags(VPP_PLATFORMS)
+def test_default(case, csc):
+  params = spec[case].copy()
+  params.update(
+    csc = csc, mcscu = mapformatu(csc),
+    format2 = csc, mformat = mapformat(params["format"]))
+  params["ofile"] = get_media()._test_artifact(
+    "{}_{format}_csc_{csc}_{width}x{height}"
+    ".yuv".format(case, **params))
+
+  call(
+    "gst-launch-1.0 -vf filesrc location={source} num-buffers={frames}"
+    " ! rawvideoparse format={mformat} width={width} height={height}"
+    " ! msdkvpp hardware=true ! video/x-raw,format=NV12"
+    " ! videoconvert ! video/x-raw,format={mcscu}"
```

Comment:
This is doing software CSC via videoconvert.  The only thing being verified on msdkvpp here is conversion to NV12.  This is not an acceptable test.  The whole purpose is to test direct input->output CSC functionality in msdkvpp only.

Reply:
ok, I will remove it. Alos, if we give an I420 input on iHD driver, the cmdline will fail, so what should we do?

Reply:
as above